### ISSUE TITLE
When help output is being piped, color information should be removed.

### ISSUE THREAD

**Issue by mrinalwadhwa** [build-trust/ockam#6949]
# Current

```
ockam project ticket --help | pbcopy
```

```
Ockam offers several pluggable enrollment protocols. This command allows project administrators to
enroll known identities or create an one-time enrollment ticket that can be used later on the end
device to enroll themselves into the project.

[1m[4mUsage:[0m [1mockam project ticket[0m [OPTIONS]

[1m[4mOptions:[0m
      [1m--identity[0m <IDENTITY_NAME>
          Run the command as the given identity name

...
```

Note the `[1m[4mUsage:[0m` characters in the output needed to add color.

# Desired

When help output is piped, all color related characters should be removed so help copyable, searchable etc.

The code for rendering help lives here:
https://github.com/build-trust/ockam/blob/develop/implementations/rust/ockam/ockam_command/src/docs.rs

**Comment by YuheMa702** [comment]
Hi! I'd like to work on this, please consider assigning it to me!

**Comment by YuheMa702** [comment]
I am a CMU student who needs to contribute to an open-source project as part of my team's final project, please consider assigning this issue to me! Thanks!

**Comment by mrinalwadhwa** [comment]
@YuheMa702 That's awesome, all yours.

Please let us know if your have any questions as you explore. You can also ask questions on the contributors discord https://discord.ockam.io

**Comment by YuheMa702** [comment]
@mrinalwadhwa Hi! I resolved this issue #6949 by making some minor changes and my PR #7085 passed all checks. Please feel free to review my PR when you are not busy. Thanks!
